lib/teselagen/api/design_client.py

```diff
@@ -191,6 +191,29 @@ def get_design(self, design_id: Union[int, str])->dict:
         out = json.loads(response["content"])
         return out
 
+    @requires_login
+    def post_design(self, design_dict:dict, allow_duplicates:bool=False):
```

Comment:
In general, I'd suggest to use more generic names (e.g. `design` instead of `design_dict`) so that modifying the type of an argument (or upgrading a method to allow multiple types) does not require modifying the name of the same in all the parts where this method is called.

Reply:
Done